### PR TITLE
Open quasm runner, to allow Q# programs to be run on the Quantum Experience

### PR DESCRIPTION
**Pull request by RolfHuisman**
Adding a quick and dirty poc to show as an example that Q# will enable people to run their programs on the IBM Quantum Experience machines as well. Original code from #55

### CODE REVIEW

File: Samples/Qiskit/QiskitQasmRunner.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+{
+    /// <summary>
+    /// Quick version to tranfer to python in a real linux environment, because Qiskit doesn't run on the windows python
+    /// </summary>
+    internal static class QiskitExecutor
+    {
+        /// <summary>
+        /// Use the linux subsystem, to run in a real linux and run the python code to execute the Quasm
+        /// </summary>
+        public static string RunQuasm(StringBuilder quasm, int qbits, string key, string backend, int shots)
```
Comment (anpaz-msft) [inline Samples/Qiskit/QiskitQasmRunner.cs]: `Quasm`… typo, should be Qasm

File: Samples/Qiskit/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+IBMQuantumExperience
```
Comment (anpaz-msft) [inline Samples/Qiskit/requirements.txt]: python3?

File: Samples/Qiskit/QiskitDriver.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        public QiskitDriver(string key): base()
+        {
+            Key = key;
+        }
+        public string Key { get; set; }
+
+        protected override IEnumerable<Result> RunOpenQasm(StringBuilder quasm, int runs)
+        {
+            Console.WriteLine("");
+            Console.WriteLine("QUASM file");
```
Comment (anpaz-msft) [inline Samples/Qiskit/QiskitDriver.cs]: qasm

File: Samples/Qiskit/QiskitDriver.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+{
+    abstract class QiskitDriver : OpenQasmDriver
+    {
+        public QiskitDriver(string key): base()
+        {
+            Key = key;
+        }
+        public string Key { get; set; }
+
+        protected override IEnumerable<Result> RunOpenQasm(StringBuilder quasm, int runs)
```
Comment (anpaz-msft) [inline Samples/Qiskit/QiskitDriver.cs]: qasm

File: Samples/Qiskit/requirements.txt (hunk 2 above)
Comment (RolfHuisman) [reply Samples/Qiskit/requirements.txt]: I'll mark it, because this is a pip requirements file. Not intended for humans :)

**Comment by martinquantum** [comment]
Thanks for contributing this substantial library to generate qasm code from Q#, Rolf!

**Pull request merged**